docs/custom_instrumentation.md

```diff
@@ -0,0 +1,171 @@
+# Custom Instrumentation
+
+This guide shows you how to add custom telemetry to your iOS app using the AWS Distro for OpenTelemetry Swift.
+
+## Custom Spans
+
+Spans track operations in your app. Use them to measure how long something takes.
+
+### Basic Span
+
+```swift
+import OpenTelemetryApi
+
+let tracer = OpenTelemetry.instance.tracerProvider.get(instrumentationName: "my-app")
```

Comment:
We support tracer/logger helpers https://github.com/aws-observability/aws-otel-swift/blob/main/Sources/AwsOpenTelemetryCore/Builder/AwsOpenTelemetryAgent.swift#L133-L139